src/service/performance-impl.js

```diff
@@ -240,24 +267,14 @@ export class Performance {
       this.flush();
     });
 
-    if (this.win.PerformanceLayoutJank) {
-      // Register a handler to record the layout jank metric when the page
-      // enters the hidden lifecycle state.
-      this.win.addEventListener(
-        VISIBILITY_CHANGE_EVENT,
-        this.boundOnVisibilityChange_,
-        {capture: true}
```

Comment:
Is the capture intended to go away?

Reply:
This block is intended to go away, the capture option is still enabled when registering the event listener in [the new codepath](https://github.com/ampproject/amphtml/pull/24346/files#diff-8ada56ee9ad7801219b44eec6947daf9R277)